extensions/amp-instagram/0.1/amp-instagram.js

```diff
@@ -108,58 +124,12 @@ class AmpInstagram extends AMP.BaseElement {
         setStyles(iframe, {
           'opacity': 1,
         });
-
-        // Hide the initial rendered image to avoid overlaying videos.
-        setStyles(this.placeholderWrapper_, {
-          'display': 'none',
-        });
+        this.togglePlaceholder(false);
```

Comment:
I agree. The only reason I thought we need this to stay is because we show the placeholder in `unlayoutCallback` and then we want it to be hidden after the iframe is rendered. But I just noticed that we actually reset `Resource.layoutCount_` to 0 - however we don't seem to reset `ElementProto.layoutCount_`  - is that a bug? Should we reset that one as well?